tox.ini

```diff
@@ -32,8 +32,8 @@ norecursedirs = .* docs requirements site-packages
 
 [testenv]
 deps =
-    django32: Django>=3.2,<4.0
-    django40: Django>=4.0,<4.1
+    django42: Django>=4.2,<5.0
+    setuptools
```

Comment:
What's the deal with requiring setuptools here, should it be in the base requirements?